### PR TITLE
Add documentation for vmong5k provider

### PR DESCRIPTION
**Pull request by jrbalderrama**

### CODE REVIEW

**Comment by jrbalderrama** [comment]
The `Vagrant` file for the `static` provider needs to be added before merging

File: docs/provider/vmong5k.rst [hunk 1]
```diff
@@ -0,0 +1,105 @@
+.. _vmong5k:
+
+Virtual Machines on Grid'5000
+=============================
+
+It is possible to deploy virtual machines on top of bare nodes of Grid'5000.
+This hybrid approach is useful to take advantage of all resouces available in
```
Comment (msimonin) [inline docs/provider/vmong5k.rst]: resouces/resources

File: docs/provider/vmong5k.rst [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+each node. In this way several virtual machines with different roles can be
```
Comment (msimonin) [inline docs/provider/vmong5k.rst]: Actually, in the current implementation vms with different won't be colocalized (enoslib limitation). But several vms with the same role can coexist.

File: docs/provider/vmong5k.rst [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+   The flavor of the resource can be set by name or using an inline description
+   with ``flavour_desc`` as is the case of the network in the example.
+
+
+Build an Image
+--------------
+
+A personalised image created and stored in Grid'5000, containing all the
+dependencis to install OpenStack in subsequent deployments, may be built
```
Comment (msimonin) [inline docs/provider/vmong5k.rst]: dependencis/dependencies

File: tests/functionnal/tests/vmong5k/basic-00.yaml [hunk 4]
```diff
@@ -1,19 +1,20 @@
...
   job_name: enos-jenkins
   type: vmong5k
   walltime: 01:00:00
-registry:
-  type: internal
 resources:
   parapluie:
     compute: 1
     control: 1
+  paravence:
```
Comment (msimonin) [inline tests/functionnal/tests/vmong5k/basic-00.yaml]: paravence/paravance

**Comment by jrbalderrama** [comment]
Typos fixed with a forced push.

**Pull request merged**